docs/pages/versions/v48.0.0/sdk/shared-element.mdx

```diff
@@ -8,6 +8,8 @@ import { APIInstallSection } from '~/components/plugins/InstallSection';
 import PlatformsSection from '~/components/plugins/PlatformsSection';
 import Video from '~/components/plugins/Video';
 
+> **info** In SDK 49 and higher, this library will no longer be available in [Expo Go](/get-started/expo-go/). You can still use it with [development builds](/develop/development-builds/introduction/).
+
 > **info** This library is listed in the Expo SDK reference because it is included in [Expo Go](/get-started/expo-go/). You may use any library of your choice with [development builds](/develop/development-builds/introduction/).
```

Comment:
I think it would make more sense to re-order these two. Maybe we can even merge them, but let's keep it clear.

```suggestion
> **info** This library is listed in the Expo SDK reference because it is included in [Expo Go](/get-started/expo-go/). You may use any library of your choice with [development builds](/develop/development-builds/introduction/).

> **info** In SDK 49 and higher, this library will no longer be available in [Expo Go](/get-started/expo-go/). You can still use it with [development builds](/develop/development-builds/introduction/).
```